pkg/provider/provider.go

```diff
@@ -7,6 +7,7 @@ import (
 var (
 	retestRegex   = regexp.MustCompile(`(?m)^/retest\s*$`)
```

Comment:
maybe we will want to  implement prows  logic https://prow.k8s.io/command-help for /retest : 

![image](https://user-images.githubusercontent.com/98980/167560801-ab6f590a-e35d-476f-bfb7-8cdd89eb7637.png)

Reply:
yeah we need to update `/retest` 
currently on retest we run all pipelinerun again

this pr adds `/test ?`
if we want we can have `/test all`

Reply:
i think we may as well do it so we don't have weird different behaviour  prow altho same keywords.

would you like to add all in this PR or later?

Reply:
`/retest` -  run only failed PRs
`/test all` - run all prs
`/test <pr-name>`  - specific pr